src/zzz_od/operation/hdd/enter_hdd_mission.py

```
@@ -34,34 +34,35 @@ def choose_chapter(self) -> OperationRoundResult:
 
         area = self.ctx.screen_loader.get_area('HDD', '章节显示')
         result = self.round_by_ocr(self.last_screenshot, self.chapter, area=area)
         if result.is_success:
-            return self.round_success(status=result.status)
+            return self.round_success()
 
         # 完成一次退出后 可能在副本列表画面
         result = self.round_by_find_area(self.last_screenshot, 'HDD', '下一步')
         if result.is_success:
-            return self.round_success(status=result.status)
+            # 在关卡选项画面稍作等待，避免过快进入下一步导致误触
```

Ruff (0.14.5)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)